docs/quickstart/apollo_2_0_hardware_system_installation_guide_v1.md

```diff
@@ -35,7 +35,7 @@
 
 # About This Guide
 
-The *Apollo 2.0 Hardware and System Installation Guide* provides the instructions to install all of the hardware components and system software for the **Apollo Project **. The system installation information included pertains to the procedures to download and install the Apollo Linux Kernel.
+The *Apollo 2.5 Hardware and System Installation Guide* provides the instructions to install all of the hardware components and system software for the **Apollo Project **. The system installation information included pertains to the procedures to download and install the Apollo Linux Kernel.
 
```

Comment:
It'd be a good idea to put all 2.5 related information in Apollo 2.5 docs instead of making changes to apollo 2.0 docs.